src/lib/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkHeader.tsx

```diff
@@ -119,24 +144,47 @@ const MyCollectionArtworkHeader: React.FC<MyCollectionArtworkHeaderProps> = (pro
   )
 }
 
-export const MyCollectionArtworkHeaderFragmentContainer = createFragmentContainer(MyCollectionArtworkHeader, {
-  artwork: graphql`
-    fragment MyCollectionArtworkHeader_artwork on Artwork {
-      artistNames
-      date
-      images {
-        height
-        isDefault
-        imageURL
-        width
+export const MyCollectionArtworkHeaderRefetchContainer = createRefetchContainer(
+  MyCollectionArtworkHeader,
+  {
+    artwork: graphql`
+      fragment MyCollectionArtworkHeader_artwork on Artwork {
+        artistNames
+        date
+        images {
+          height
+          isDefault
+          imageURL
+          width
+          internalID
+          imageVersions
+        }
+        internalID
+        slug
+        title
+      }
+    `,
+  },
+  graphql`
+    query MyCollectionArtworkHeaderRefetchQuery($artworkID: String!) {
+      artwork(id: $artworkID) {
+        artistNames
+        date
+        images {
+          height
+          isDefault
+          imageURL
+          width
+          internalID
+          imageVersions
+        }
         internalID
+        slug
+        title
```

Comment:
We can use the fragment defined above instead of duplicating the same fields here
```suggestion
        ... MyCollectionArtworkHeader_artwork
```